src/App/Hovedside/TjenesteBoksContainer/TjenesteBoksBanner/TjenesteBoksBanner.less

```diff
@@ -2,7 +2,7 @@
 
 .tjeneste-boks-banner {
     display: flex;
-    background-color: @navBlaLighten80;
+    background-color: #cce1ff;
```

Comment:
Kan være greit å finne navnet på fargen (er vel en variant av `@navBla` eller noe)

Reply:
Jeg tror det er `@navBlaDarken40`. https://github.com/navikt/bedriftsmeny/blob/db519d9652a21044321ed5ed239f6a060bc7a42c/src/index.less#L9

Reply:
So close! CCE1FF vs CCE1F3.. Ser helt likt ut for folk flest men..

Reply:
Går for @navBlaDarken80. Vi endrer på headeren også.